doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -326,234 +345,308 @@ Per-worker relations expose the same data as their global counterparts, but have
-| `export_id`  | [`text`]     | The ID of the dataflow export. Corresponds to [`mz_compute_exports.export_id`](#mz_compute_exports).                                                                                                                                                 |
-| `import_id`  | [`text`]     | The ID of the dataflow import. Corresponds to either [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources) or [`mz_catalog.mz_tables.id`](../mz_catalog#mz_tables) or [`mz_catalog.mz_materialized_views.id`](../mz_catalog#mz_materialized_views). |
-| `delay_ns`   | [`bigint`]   | The upper bound of the bucket in nanoseconds.                                                                                                                                                                                                        |
-| `count`      | [`bigint`]   | The (noncumulative) count of delay measurements in this bucket.                                                                                                                                                                                      |
+<!-- RELATION_SPEC mz_internal.mz_compute_delays_histogram -->
+| Field        | Type        | Meaning                                                                                                                                                                                                                                              |
+| ------------ |-------------| --------                                                                                                                                                                                                                                             |
+| `export_id`  | [`text`]    | The ID of the dataflow export. Corresponds to [`mz_compute_exports.export_id`](#mz_compute_exports).                                                                                                                                                 |
+| `import_id`  | [`text`]    | The ID of the dataflow import. Corresponds to either [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources) or [`mz_catalog.mz_tables.id`](../mz_catalog#mz_tables) or [`mz_catalog.mz_materialized_views.id`](../mz_catalog#mz_materialized_views). |
+| `delay_ns`   | [`uint8`]   | The upper bound of the bucket in nanoseconds.                                                                                                                                                                                                        |
+| `count`      | [`numeric`] | The (noncumulative) count of delay measurements in this bucket.                                                                                                                                                                                      |
```

Comment:
It is at least surprising to see `count` having type `numeric`... 🤔 
(Though, I understand this change just aligns documentation with reality.)

A bit below, `sent` and `received` (messages) are a similar case.

Reply:
@antiguru to file a bug.

Reply:
#20522